src/imp/tracer_imp.js

```diff
@@ -883,11 +883,13 @@ export default class TracerImp extends EventEmitter {
         let clockOffsetMicros = this._clockState.offsetMicros();
 
         // Diagnostic information on the clock correction
-        this.logStable('cr/time_correction_state', {
-            offset_micros  : clockOffsetMicros,
-            active_samples : this._clockState.activeSampleCount(),
-            ready          : clockReady,
-        });
+        if (this._options.debug) {
```

Comment:
The reporting test launches a child process as part of the test. In other words, there will be two separate instances of Node code running (the host and the child process) -- but they are part of one test. Perhaps that's what you were seeing?